### PR TITLE
added user test

### PR DESCRIPTION
**Pull request by SafnetCo2**
## Description

Please include a summary of the changes and the related issue.
Add any screenshots if a change has been made on the client side.

## Type of change

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: server/app/routes.py [hunk 1]
```diff
@@ -43,54 +43,48 @@ def get_user(user_id):
...
+
         db.session.add(new_user)
         db.session.commit()
+
+        response_data = {
+            'status': 'Success',
+            'message': 'User created successfully.',
+            'data': new_user.to_dict()
+        }
+        print(response_data)
```
Comment (otsembo) [inline server/app/routes.py:74]: Once you finish debugging, you can remove the print statements.

**Review by otsembo — APPROVED** [review]
Good Job @SafnetCo2

**Comment by SFrasia** [comment]
Good

**Comment by otsembo** [comment]
@SFrasia I can see this was closed instead of merged. Is this the correct scenario or was it done mistakenly?

**Comment by SFrasia** [comment]
@otsembo it was by mistake

**Comment by otsembo** [comment]
Okay. You can reopen it and then perform a merge @SFrasia

**Pull request closed without merging**